packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -0,0 +1,200 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  MouseEventHandler,
+  ReactElement,
+  memo,
+  useMemo,
+  ReactNode,
+  HTMLAttributes,
+} from "react";
+import {
+  Card as MuiCard,
+  CardActionArea as MuiCardActionArea,
+} from "@mui/material";
+import styled from "@emotion/styled";
+
+import { Button } from ".././Button";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from ".././OdysseyDesignTokensContext";
+import { Heading5, Paragraph, Subordinate, Support } from ".././Typography";
+import { Box } from ".././Box";
+
+export const APP_TILE_IMAGE_HEIGHT = "64px";
+
+export type AppTileProps = {
+  // Text that appears in the upper right corner of the tile
+  auxiliaryText?: string;
+  // Arbitrary content to render underneath any other tile content
+  children?: ReactNode;
+  // A string description
+  description?: string;
+  // An image or icon at the top of the tile
+  image?: ReactElement;
+  // Event handler for when the user clicks the tile
+  onClick: MouseEventHandler;
+  // An 'eyebrow' of text above the title
+  overline?: string;
+  // A string for the tile title
+  title?: string;
+} & (
+  | {
+      // Event that fires when the user clicks the action button in the upper-right corner
+      // If this isn't set, the other action props can't be set either
+      onActionClick: MouseEventHandler;
+      // The ID of the element which the button controls (for instance, a drawer or dialog), if any.
+      actionAriaControls?: HTMLAttributes<HTMLElement>["aria-controls"];
+      // Should be true if the button controls a popup element such as a Drawer or Dialog
+      actionAriaHasPopup?: HTMLAttributes<HTMLElement>["aria-haspopup"];
```

Comment:
This can be other values besides `true` depending on the menu type. If this opens a dialog the value would be `aira-haspopup="dialog"`

Reply:
Oh, right, just updated the type.